fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -445,6 +445,9 @@ struct BlobCipherEncryptHeaderRef {
 		}
 	}
 
+	uint8_t* getIV() const;
+	std::tuple<BlobCipherDetails, BlobCipherDetails> getCipherDetails() const;
```

Comment:
We should document that this method returns tuple<text cipher key, header cipher key>

Reply:
Maybe it would be more clear if we had a type of TextAndHeaderCipherDetails, that way we wont have to always remember which is which

Reply:
@sfc-gh-nwijetunga  this was just for display. I'm working on a patch that adds the helper methods to unblock you. Shall post soon (in an hour or so)

Reply:
Sure just posting my thoughts as i'm going through it